notification_delivery/config.py

```diff
@@ -29,6 +29,30 @@ class Config:
     DEBUG = False
     LIGHT_ID = '1'
 
+    # Mappings between Google Cloud alerting policy names
+    # and philips hue (color) values (0 to 65535). Each
```

Comment:
I'd assume that the colors are some sort of standardized system? I think HSB? Maybe name that system here and elsewhere in the documentation?

Reply:
Done